src/client/Microsoft.Identity.Client/OAuth2/OAuth2Client.cs

```diff
@@ -16,22 +16,30 @@
 using System.Threading.Tasks;
 using Microsoft.Identity.Client.Instance.Discovery;
 using Microsoft.Identity.Client.TelemetryCore.Internal;
+using System.Linq;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
 
 namespace Microsoft.Identity.Client.OAuth2
 {
     internal class OAuth2Client
     {
+        private const string DeviceAuthHeaderName = "x-ms-PKeyAuth";
```

Comment:
There are some classes with constants around, please move these constants there, preferably in a new class, smth like PKAuthConstants.DeviceAuthHeaderName